Cabal/Distribution/PackageDescription/Check.hs

```diff
@@ -1904,3 +1903,25 @@ fileExtensionSupportedLanguage path =
     extension = takeExtension path
     isHaskell = extension `elem` [".hs", ".lhs"]
     isC       = isJust (filenameCDialect extension)
+
+-- Don't use System.FilePath.isRelative. The System.FilePath exists in two
```

Comment:
If the answer to my comment above is positive, I suggest moving these two functions to `Distribution.Simple.Utils`.